101-machine-learning-compute-create-computeinstance/azuredeploy.json

```diff
@@ -95,17 +71,22 @@
         {
             "type": "Microsoft.MachineLearningServices/workspaces/computes",
             "name": "[concat(parameters('workspaceName'), '/', parameters('computeName'))]",
-            "apiVersion": "2020-06-01",
+            "apiVersion": "2020-09-01-preview",
             "location" : "[parameters('location')]",
             "properties": {
                 "computeType": "ComputeInstance",
                 "properties":
                 {
                     "vmSize" : "[parameters('vmSize')]",
-                    "subnet" : "[if(and(not(empty(parameters('vnetResourceGroupName'))),not(empty(parameters('vnetName'))),not(empty(parameters('subnetName')))), variables('subnet'), json('null'))]",
-                    "sshSettings" : "[if(equals(parameters('sshPublicAccess'), 'Enabled'), variables('sshSettings'), json('null'))]"
+                    "subnet" : "[if(and(not(empty(parameters('vnetResourceGroupName'))),not(empty(parameters('vnetName'))),not(empty(parameters('subnetName')))), variables('subnet'), json('null'))]",                    
+                    "personalComputeInstanceSettings": {
+                    "assignedUser": {
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
Done